src/project.rs

```diff
@@ -48,8 +49,13 @@ fn default_include() -> Vec<PathBuf> {
 
 impl Project {
     pub fn save(&self) -> Result<(), Error> {
-        let mut out = File::create(crate::HEMTT_FILE)?;
-        out.write_fmt(format_args!("{}", serde_json::to_string_pretty(&self)?))?;
+        let file = path().unwrap();
```

Comment:
This panics instead of printing pretty error from `path()`.
```suggestion
        let file = path().print_error(true);
```